app/controllers/concerns/curriculum/rateable.rb

```diff
@@ -35,7 +35,7 @@ def comment
       response = client.update_rating(
         id: request[:rating_id],
         key: :comment,
-        value: request[:comment],
+        value: request[:comment].gsub('"', '&quote;'),
```

Comment:
it would be more flexible to use something like `CGI.escapeHTML` (https://ruby-doc.org/stdlib-3.1.0/libdoc/cgi/rdoc/CGI/Escape.html)...

however maybe a question for @maddyjoe is whether this is ok? since there'll now be entities in the comments which will make them harder to read, perhaps we'd be better stripping quotes in a way that doesn't add alter the string

Reply:
could you also add some tests around this please